event/block.go

```diff
@@ -39,14 +39,15 @@ func (e ErrEventBlocked) Error() string {
 }
 
 type Block struct {
-	ID        bson.ObjectId `bson:"_id,omitempty"`
-	StartTime time.Time
-	EndTime   time.Time `bson:"endtime,omitempty"`
-	KindName  string
-	OwnerName string
-	Target    Target `bson:"target,omitempty"`
-	Reason    string
-	Active    bool
+	ID          bson.ObjectId `bson:"_id,omitempty"`
+	StartTime   time.Time
+	EndTime     time.Time `bson:"endtime,omitempty"`
+	KindName    string
+	OwnerName   string
+	Target      Target            `bson:"target,omitempty"`
+	ExtraFields map[string]string `bson:"extraFields,omitempty"`
```

Comment:
@morpheu could you Rename ExtraFields to Restrictions or Conditions ?

Reply:
Sure! Done :)